client/ui/components/pages/AddonGroupsListPage.jsx

```diff
@@ -2,12 +2,14 @@ import React from 'react';
 
 import Paper from 'material-ui/Paper';
 
-import AddonGroupsList from '../AddonGroupsList';
+import AddonGroupsList from '../addon-groups/AddonGroupsList';
+import QueryAddonGroups from '../data/QueryAddonGroups';
 
 
 function AddonGroupsListPage() {
   return (
     <Paper zDepth={2} className="page">
+      <QueryAddonGroups />
```

Comment:
Why put the query here instead of with the component that depends on fetching the groups?